cmd/metrics/cmd/mock_dest_store.go

```diff
@@ -25,7 +25,7 @@ func (mds *mockDestStore) String() string {
 
 func (mds *mockDestStore) Has(ctx context.Context, key string) (bool, error) {
 	// detect and respond to RepoExists() call
-	if strings.HasPrefix(key, "repos/") && strings.HasSuffix(key, "/repo.json") {
+	if strings.HasPrefix(key, "repos/") && strings.HasSuffix(key, "/repo.yaml") {
```

Comment:
how about repos created with pre-v2 datamon? should they be migrated or do we want to had some compatibility path here?

Reply:
We will move the data over. It will be separate buckets.

Reply:
ok as per our discussion